src/task_types/fuzzer_tasks.rs

```diff
@@ -216,27 +217,24 @@ impl GenFuzzerService for GenFuzzerServiceImpl {
         let task_name = &params.task_name;
         let mut sub_tasks: Vec<EvgTask> = vec![];
         if params.is_multiversion() {
+            if params.multiversion_generate_tasks.is_empty() {
```

Comment:
This works & is okay, but I think the more rust idiomatic way to do this is to make
```rust
pub multiversion_generate_tasks: Vec<MultiversionGenerateTaskConfig>,
```
into
```rust
pub multiversion_generate_tasks: Option<Vec<MultiversionGenerateTaskConfig>>,
```
And then unwrap the variable so the panic is built-in.